cobra.go

```diff
@@ -205,3 +206,17 @@ func stringInSlice(a string, list []string) bool {
 	}
 	return false
 }
+
+// Er prints the msg with the prefix 'Error:' and exits with error code 1. If the msg is nil, it does nothing.
+func Er(msg interface{}) {
```

Comment:
Why are we exporting this? Looks like we are only using it internally.
Also `Er` doesn't _mean_ anything. Can we rename it to something a bit more telling?

Reply:
We are exporting it because it is used in different internal packages. One is cobra itself, the other ones are `cmd` and `doc`. These do not inherit functions from cobra. Hence, we need to export it if we want to reuse it without rewriting the same function three times. See review comment by @wfernandes above.

Regarding the name `er`, it's been in the codebase since @bogem added it three years ago: https://github.com/spf13/cobra/blame/master/cobra/cmd/helpers.go#L62. I just reused it first and then uppercased it. Last, after @wfernandes suggestion, I replaced it with `assertNoErr` in `*_test.go` sources. See also https://github.com/spf13/cobra/commit/21cab29ef902d36da074e8c940437c9f536b2283. Nevertheless, I'm ok with renaming it if you tell me what name to use instead.

Reply:
So IMO, if we are to export this method, it needs a better name. I wouldn't be a stickler if it was internal to the code base. But exporting it means it will be public facing in the docs so we should attempt to name it a bit better.
@wfernandes opinion?

Reply:
Thinking about it, I agree with @jharshman. Maybe we can call it `PrintErr`. WDYT?

Reply:
I renamed `Er` to `CheckErr`, as @marckhouzam wrote in https://github.com/spf13/cobra/blame/master/shell_completions.md#L90. Although I think it might have been a typo when copying it from somewhere else, the name feels good.

I also removed the indirections for both `Er` and `WrStringAndCheck`, as @wfernandes commented above.

Reply:
> I renamed `Er` to `CheckErr`, as @marckhouzam wrote in https://github.com/spf13/cobra/blame/master/shell_completions.md#L90

That was a copy paste from the old doc 😁.  I believe`util.CheckErr()` is code from `kubectl`. It's a good name in my opinion also.